### ISSUE TITLE
"make html" failing in repo

### ISSUE THREAD

**Issue by michaeljones** [michaeljones/breathe#732]
If I run `make html`, I get:
```
Running Sphinx v4.1.2
Original exception:
Traceback (most recent call last):
  File "/home/michael/root/projects/breathe/venv/lib/python3.8/site-packages/sphinx/registry.py", line 429, in load_extension
    mod = import_module(extname)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'breathe'

Exception occurred while building, starting debugger:
Traceback (most recent call last):
  File "/home/michael/root/projects/breathe/venv/lib/python3.8/site-packages/sphinx/registry.py", line 429, in load_extension
    mod = import_module(extname)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'breathe'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/michael/root/projects/breathe/venv/lib/python3.8/site-packages/sphinx/cmd/build.py", line 276, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/home/michael/root/projects/breathe/venv/lib/python3.8/site-packages/sphinx/application.py", line 237, in __init__
    self.setup_extension(extension)
  File "/home/michael/root/projects/breathe/venv/lib/python3.8/site-packages/sphinx/application.py", line 393, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/michael/root/projects/breathe/venv/lib/python3.8/site-packages/sphinx/registry.py", line 432, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname,
sphinx.errors.ExtensionError: Could not import extension breathe (exception: No module named 'breathe')
> /home/michael/root/projects/breathe/venv/lib/python3.8/site-packages/sphinx/registry.py(432)load_extension()
-> raise ExtensionError(__('Could not import extension %s') % extname,
(Pdb) --KeyboardInterrupt--
(Pdb)
```
It is fixed if I change:
https://github.com/michaeljones/breathe/blob/f0b8ecb992325785575b6f894783a8ae43680bf3/documentation/source/conf.py#L82
to be `../../` instead of `../`.

Is anyone else seeing that? Maybe Sphinx changed its concept of a current working directory at some point? I'm not sure. Maybe no one else is seeing it.

**Comment by jakobandersen** [comment]
I see the same problem and indeed the extra ``../`` fixes it. The reason why I probably haven't encountered it before is that I have installed the package in edit mode (``pip install -e .`` in the root) so the import would succeed anyway. Not sure why this isn't caught by the CI tests.

**Comment by michaeljones** [comment]
Thanks for double checking. I thought maybe you'd found other ways of doing it. 

CI seems to manipulate the PYTHONPATH: https://github.com/michaeljones/breathe/blob/master/.github/workflows/documentation.yml#L25-L28

Maybe that's why it is ok?

**Comment by jakobandersen** [comment]
Ah, yes, that could explain it.

**Comment by michaeljones** [comment]
This has been fixed in #734
